spec/requests/posts_spec.rb

```diff
@@ -2,34 +2,34 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET index' do
-    before(:example) { get 'htttp://localhost:3000/posts/index' }
+    before(:example) { get user_posts_path(2) }
 
     it 'is a success' do
       expect(response).to have_http_status(:ok)
     end
 
     it "renders 'index' template" do
-      expect(response).to render_template('index')
-    end
-
-    it "renders 'index' template with template in body" do
-      expect(response.body).to match('Posts')
+      expect(response).to render_template(:index)
     end
   end
 
   describe 'GET show' do
-    before(:example) { get 'htttp://localhost:3000/posts/show' }
+    Post.create(
+      user_id: 2,
+      title: 'Wired',
+      text: 'What does the economy have to do with technological innovations of the future?',
+      likes_counter: 0,
+      comments_counter: 0
+    )
+
+    before(:example) { get user_post_path(2, 1) }
 
     it 'is a success' do
       expect(response).to have_http_status(:ok)
     end
 
     it "renders 'show' template" do
-      expect(response).to render_template('show')
-    end
-
-    it "renders 'show' template with template in body" do
-      expect(response.body).to match('Show')
+      expect(response).to render_template(:show)
     end
```

Comment:
- [ ] They are 4 tests that are failing in this file please fix them
![test](https://user-images.githubusercontent.com/44745855/156181527-bd88f15a-ea24-4c89-a7a2-1fec1803e21e.png)